utils/difference.js

```diff
@@ -14,15 +14,14 @@ function doDifferenceCalculations(row) {
  * @param{row} - The row to do calculations for
  */
 function calculateDifferences(row) {
-  const updatedRow = row;
-  if (exists(updatedRow.sum) && exists(updatedRow.comparison_sum)) {
-    updatedRow.difference_sum = executeFormula('delta', [updatedRow.sum, updatedRow.comparison_sum]);
+  if (exists(row.sum) && exists(row.comparison_sum)) {
+    row.difference_sum = executeFormula('delta', [row.sum, row.comparison_sum]);
```

Comment:
There's some weird linting rule _against_ mutating function params, but I don't know why it's a rule and also, if the param is an _object_ and you only mutate properties of that object (which is all you're doing), I think the rule doesn't apply.